src/styles/blocks/style.scss

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background: #000;
+}
+
+.container {
+  width: calc(8rem + 15px);
+  height: calc(8rem + 15px);
+  border-radius: 50%;
+  background: linear-gradient(-230deg, rgb(100, 6, 252), green, blue);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.clock {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 8rem;
+  height: 8rem;
+  background: #000;
+  border-radius: 50%;
+}
+
+span {
```

Comment:
it's unclear what is it. set some class like `clock__center` and use class selector in CSS instead of `span`